Dockerfile

```diff
@@ -1,19 +1,20 @@
-FROM python:3.8-slim
+FROM nvcr.io/nvidia/tensorflow:22.10.1-tf2-py3
 
-ENV PYTHONUNBUFFERED 1
-ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED=1 
+ENV PYTHONDONTWRITEBYTECODE=1 
+ENV DOCTR_CACHE_DIR=/app/.cache 
+ENV PATH=/app/venv/bin:$PATH
 
+WORKDIR /app
 
-COPY ./pyproject.toml /tmp/pyproject.toml
-COPY ./README.md /tmp/README.md
-COPY ./setup.py /tmp/setup.py
-COPY ./doctr /tmp/doctr
+COPY . .
```

Comment:
@ffalkenberg what's the reason to copy everything ?
I think we should exclude parts like the docs folder, and so on :)

Reply:
@felixT2K Thank you for pointing this out.

Here's the breakdown from the `du` output for the `doctr` directory:

```
57M     doctr/.git
100K    doctr/.github
92K     doctr/api
28K     doctr/demo
4.0M    doctr/docs
1.3M    doctr/doctr
8.0K    doctr/notebooks
268K    doctr/references
52K     doctr/scripts
348K    doctr/tests
63M     doctr/
```

From the output, we can observe that while certain directories like `doctr/docs` have a relatively larger size (4.0M) compared to others, the potential memory savings by excluding them would still be negligible in my opionion, but that would be your choice. 

The inclusion of these directories was a simpler and more direct approach, and I specifically wanted to retain directories such as `demo` and `api` based on my needs.